AUTHORS.md

```diff
@@ -0,0 +1,146 @@
+Lv Jiawei                                @MIBc
+maao                                     @420691301
+Mark Fermor                              @markfermor
+Mark Petrovic                            @ae6rt
+Martin Evgeniev                          @suizman
+Mat Meredith                             @MatMeredith
+Matt Dupre                               @matthewdupre
+Matt Kelly                               @mattkelly
+Matt Leung                               @mgleung
+Maxim Ivanov                             @redbaron
+Maxime Guyot                             @Miouge1
+Michael Dong                             @michaeldongvrviu
+Michael Stowe                            @mikestowe
+Mike Frisch                              @EmmEff
+Mike Scherbakov                          @mihgen
+Mike Spreitzer                           @MikeSpreitzer
+Muhammad Saghir                          @msagheer
+Neil Jerram                              @neiljerram
+Nic Doye                                 @nicdoye
+Nick Bartos                              @nbartos
+Nick Wood                                @nwoodmsft
+Noah Treuhaft                            @nwt
+Paul Tiplady                             @paultiplady
+Pavel Khusainov                          @alvelcom
+Peter White                              @plwhite
+Pike                                     @pikeszfish
+Qiu Yu                                   @unicell
+Rahul Krishna Upadhyaya                  @rakrup
+Renan Gonçalves                         @renan
+Ricardo Katz                             @rikatz
+Rob Brockbank                            @robbrockbank
+Roma Sokolkov                            @r7vme
+Rush Tehrani                             @rushtehrani
+Ryan Zhang                               @ry4nz
+Saeid                                    @sudomakeinstall2
+Saurabh Mohan                            @saumoh
+Seth                                     @sethmccombs
+Shatru                                   @shatrugna
+Shaun Crampton                           @fasaxc
+SONG JIANG                               @song-jiang
+Spike Curtis                             @spikecurtis
+Stephen Schlie                           @heschlie
+Suraj Narwade                            @surajnarwade
+Tamal Saha                               @tamalsaha
+Timo Beckers                             @ti-mo
+Timothy Briggs                           @TrimBiggs
+Tom Denham                               @tomdee
+Tomas                                    @ToroNZ
+Tomas Mazak                              @tomas-mazak
+Vincent Schwarzer                        @VincentS
+Wei Kin Huang                            @weikinhuang
+Wietse Muizelaar                         @wmuizelaar
+YAMAMOTO Takashi                         @yamt
+Yecheng Fu                               @cofyc
+Yumo Yang                                @elementyang
+Zhaole Yang                              @ijumps
+Àbéjídé Àyodélé                   @bjhaid
```

Comment:
Well, 7 spaces missing, and 7 non-ASCII characters in Àbéjídé's name, and the encoding is UTF-8...  Looks like the formatting magic of `{0:<40}` is wrongly counting bytes instead of characters.

Try instead `fmt.Printf("%40v@%v", name, login)`?  https://golang.org/pkg/fmt/ explicitly says:

> Width and precision are measured in units of Unicode code points, that is, runes. (This differs from C's printf where the units are always measured in bytes.) 

which sounds like what we want.

Reply:
Seems Python3 handles this a lot better.

But, it's also moot since I decided to add in markdown table support, meaning we wouldn't hit this issue anyway.